src/pages/[platform]/build-a-backend/index.mdx

```diff
@@ -1,24 +1,29 @@
 import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+import { getChildPageNodes } from '@/utils/getChildPageNodes';
+import directory from 'src/directory/directory.json';
 
 export const meta = {
   title: 'Build a backend',
   description: 'Build a backend',
-  platforms: ['android', 'javascript']
+  platforms: ['android', 'javascript'],
+  route: '/[platform]/build-a-backend'
```

Comment:
In order to get the component to work with SSG, we'll have to add the `route` property here so that we can locate it inside `directory.json`. This is because `getStaticProps` doesn't have access to the full route and you can't use `useRouter` in there either